components/Users/Memberships/index.js

```diff
@@ -120,7 +122,68 @@ const getState = (membership) => {
   return 'aktiv'
 }
 
+
+const SET_AUTO_PAY = gql`
+  mutation setMembershipAutoPay($id: ID!, $autoPay: Boolean!) {
+    setMembershipAutoPay(id: $id, autoPay: $autoPay) {
+      id
+      autoPay
+    }
+  }
+`
+
+function AutoPayToggle (membership) {
```

Comment:
We generally use arrow function everywhere (no `this` scope) in the frontend:
```suggestion
const AutoPayToggle = (membership) => {
```